app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -284,7 +284,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
   };
 
   $scope.isDetectionEnabled = function() {
-    return ($scope.emsCommonModel.metrics_selection == "hawkular" && $scope.emsCommonModel.ems_controller == "ems_container") &&
```

Comment:
Change `==` to `===`

Reply:
👍 done ! thanks, using too many languages at once :-)

p.s.
can't we ask codeclimate/rubocop to worn on this things ?

Reply:
I'm surprised codeclimate does not call us out on those `==`

Your other option is to install the codeclimate plugin which will make these things visible as you browse your JS code in github.

Reply:
👍 I will also look for a better js linter for my editor ...